README.rst

```diff
@@ -101,6 +101,17 @@ Once installed, you can send telemetry to Application Insights. Here are a few s
     tc.track_trace('My trace with context')
     tc.flush()
 
+**Establishing correlation between telemetry objects**
+
+context field called operation_id can be set to associate telemetry items.

```

Comment:
Can you please add a note that will tell not to re-use telemetry client in multy-threaded way

Reply:
Can't we reuse the telemetry client if we reset the operation_id for future requests?
Either set the operation_id to null (after use) or some other value in future request.

Reply:
yes. the issue is to use it in parallel threads and how to avoid the concurrency issues. In any casde I'd appreciate if you can leave a comment here about it.

Reply:
updated